tools/container/runtime/containerd/containerd.go

```diff
@@ -164,3 +169,13 @@ func (o *Options) runtimeConfigOverride() (map[string]interface{}, error) {
 
 	return runtimeOptions, nil
 }
+
+// getContainerdConfigCommand returns a string slice which contains the CLI args to retrieve the current runtime configuration
+func getContainerdConfigCommand(o *container.Options) []string {
+	var cliArgs []string
+	if o.HostRootMount != "" {
+		cliArgs = append(cliArgs, "chroot", o.HostRootMount)
+	}
+	cliArgs = append(cliArgs, "crio", "status", "config")
```

Comment:
This is incorrectly using the crio command.

Reply:
thanks for catching that